src/js/components/Select/stories/select.stories.js

```diff
@@ -73,31 +73,111 @@ class SimpleSelect extends Component {
   }
 }
 
-const DEFAULT_OPTIONS = [];
-for (let i = 1; i <= 200; i += 1) DEFAULT_OPTIONS.push(`option ${i}`);
+const defaultOptions = [];
+const objectOptions = [];
+for (let i = 1; i <= 200; i += 1) {
```

Comment:
IMO 200 is not a good enough test for "lots of options". I'm having issues when I bump this to 3000. Now this may be fixed when InfiniteScroll allows virtualization (only render what's on the screen), but with 3000 when I rapidly scroll to the bottom, it keeps getting "stuck" very easily (i.e. not rendering any more items) until i scroll up a bit, then scroll back down again.

Reply:
This will be addressed when the improvements to InfiniteScroll are done.